client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -285,22 +309,31 @@ const RecurringPaymentsPlanAddEditModal = ( {
 				</FormFieldset>
 				<FormFieldset>
 					<FormToggle onChange={ handleMultiplePerUser } checked={ editedMultiplePerUser }>
-						{ translate( 'Allow the same customer to sign up multiple times to the same plan.' ) }
+						{ isPayments
+							? translate(
+									'Allow the same customer to purchase or sign up to this plan multiple times.'
+							  )
+							: translate( 'Allow the same customer to sign up multiple times to the same plan.' ) }
 					</FormToggle>
 				</FormFieldset>
 			</>
 		);
 	};
 
 	const renderEmailTab = () => {
+		const isPayments = isEnabled( 'earn/rename-payment-blocks' );
 		return (
 			<>
 				<FormFieldset>
 					<h6 className="memberships__dialog-form-header">{ translate( 'Posts via email' ) }</h6>
 					<p>
-						{ translate(
-							'Allow members of this recurring payment plan to opt into receiving new posts via email.'
-						) }{ ' ' }
+						{ isPayments
+							? translate(
+									'Allow members of this payment plan to opt into receiving new posts via email.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.